truss/cli/cli.py

```diff
@@ -582,12 +582,12 @@ def push_chain(
             remote=remote,
         )
         service = chains_remote.push(entrypoint_cls, options)
-        assert isinstance(service, chains_remote.BasetenChainService)
 
     console.print("\n")
```

Comment:
why do we have this? I was always confused why there is a gap after i press enter

Reply:
Does it bug you a lot? I found it somehow easier to read in my console history, when running several commands in succession. But I'm also down to remove it.

Reply:
It's fine. i just am curious why it was there. it doesn't both me too much.